test/routes/api/tasks/tasks.test.ts

```diff
@@ -629,7 +629,7 @@ describe('Tasks api (logged user only)', () => {
         const largeTestImagePath = path.join(tmpDir, 'large-test-image.jpg')
 
         const largeBuffer = Buffer.alloc(1024 * 1024 * 1.5, 'a') // Max file size in bytes is 1 MB
-        fs.writeFileSync(largeTestImagePath, largeBuffer)
+        fs.writeFileSync(largeTestImagePath, largeBuffer, { mode: 0o600 }) // 0600 permissions (read/write for owner only)
```

Comment:
```suggestion
        /**
         * Limit file permissions to read/write for the owner only,
         * as permissions should follow the principle of least privilege.
         * @see {@link https://learn.snyk.io/lesson/insecure-temporary-file/?ecosystem=javascript | Insecure temporary file}
         */
        fs.writeFileSync(largeTestImagePath, largeBuffer, { mode: 0o600 }) // 0600 permissions (read/write for owner only)
```

@jean-michelet this okay?

Reply:
Yes :+1: